src/metabase/util/ssh.clj

```diff
@@ -0,0 +1,95 @@
+(ns metabase.util.ssh
+  (:require [clojure.tools.logging :as log]
+            [metabase.util :as u])
+  (:import com.jcraft.jsch.JSch))
+
+(def default-ssh-timeout 30000)
+
+(defn start-ssh-tunnel
```

Comment:
Just a heads up I have a linter set up so every public var has to have a docstring. (Trying to keep things ridiculously-well documented to encourage potential contributors)

So you'll need to either add a quick blurb explaining what the functions do and other vars are for or otherwise make things `^:private`